src/LandingPage/LandingPage.test.js

```diff
@@ -12,10 +12,11 @@ describe('LandingPage', () => {
  })
 
   it('should render a greeting text and array of images', () => {
+    // how do we test for the larger welcoming text on landing page?
```

Comment:
If it's what renders by default we should be able to get it with getByText without doing anything fancy? Maybe we have to do a mocked fetch that returns an empty set of data to simulate it?